packages/e2e-tests/src/specs/dashboard/settings/adminUser/shoppingProvider.js

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import {
+  visitSettings,
+  withExperimentalFeatures,
+} from '@web-stories-wp/e2e-test-utils';
+
+/**
+ * Internal dependencies
+ */
+import { shoppingProviderDropdownSelector } from '../../../../utils';
+
+describe('Admin User', () => {
```

Comment:
Let's have a better description for this. I know this done other places, but these should changed as well.